services.html

```diff
@@ -1,38 +1,230 @@
+                      premier niche markets. Professionally cultivate one-tone
+                      customer service with robust ideas. Dynamically innovate
+                      resource-leving custoemr service for state of the art
+                      customer service.
+                    </p>
+                    <br />
+                    <p>
+                      Phosfluorescently engage worldwide methodologies with
+                      web-enabled technology. Interactively coordinate proactive
+                      e-commerce via process-centric "outside the box" thinking.
+                      Completely pursue scalable customer service through
+                      sustainable potentialities.
+                    </p>
+                  </div>
+                </div>
+                <img
+                  src="/img/services/services-tab-sustainability-img.png"
+                  alt=""
+                />
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
 
-  Completely synergize resource taxing relationships via premier niche markets. Professionally cultivate one-to-one customer service with robust ideas. Dynamically innovate resource-leveling customer service for state of the art customer service.  Phosfluorescently engage worldwide methodologies with web-enabled technology. Interactively coordinate proactive e-commerce via process-centric "outside the box" thinking. Completely pursue scalable customer service through sustainable potentialities.
+      <!-- FOOTER CONTENT -->
 
+      <section class="bottom-content">
+        <section class="forum">
+          Interested in starting a project? <br />
+          Let’s talk: <br />
+          <input type="email" placeholder="Enter Email" class="email-stuff" />
+          <figcaption>
+            We'll never share your email with anyone else.
+          </figcaption>
+        </section>
 
+        <div class="new-york">
+          <h1>New York</h1>
+          <br />
+          <p>
+            123 Lane <br />
+            Suite 100 <br />
+            Albany, NY 12345 <br />
+            202 555 0144
+          </p>
+        </div>
 
--->
+        <div class="florida">
+          <h1>Florida</h1>
+          <br />
+          <p>
+            Ocean Drive <br />
+            Suite 201 <br />
+            Orlando, FL 22345 <br />
+            502 555 0144
+          </p>
+        </div>
 
-	<script src="js/index.js"></script>
-</body>
-</html>
\ No newline at end of file
+        <div class="california">
+          <h1>California</h1>
+          <br />
+          <p>
+            Mountain Street <br />
+            Suite 105 <br />
+            San Diego, CA 22345 <br />
+            702 555 0144
+          </p>
+        </div>
+      </section>
+      <!-- copyright footer -->
+      <footer>Copyright © 2018 Smith and Jones</footer>
+    </div>
+    <script src="/js/tabs.js"></script>
```

Comment:
Bad path to JS script file. Because of this functionality of tabs was not working. In real life environment nobody would accept such project or spend time to look through whats wrong. You need to be more accurate and debug Your code.